blog/2023-03-18-tauri-2-0-0-alpha-4.mdx

```diff
@@ -58,6 +58,13 @@ cargo add tauri-build@2.0.0-alpha.2 --build
 cargo install tauri-cli --version "^2.0.0-alpha"
 ```
 
+Recreate the mobile projects to use the new features:
+
+```shell
+rm -r src-tauri/gen
+tauri android init
+tauri ios init
+
```

Comment:
This might be a bit late but did this need the ending back ticks?

Reply:
Yep it did. It was added in a follow-up pr shortly after.